src/nodes/materials/MeshBasicNodeMaterial.js

```diff
@@ -28,7 +28,11 @@ class MeshBasicNodeMaterial extends NodeMaterial {
 
 	setupNormal() {
 
-		transformedNormalView.assign( normalView ); // see #28839
+		if ( this.envNode || this.envMap ) {
```

Comment:
Unfortunately, this does not work. A cube texture node can be directly assigned to `colorNode` so this `if` statement is insufficient.

Reply:
Thanks for trying!  :-)

Reply:
We are getting there! 💪